docs/developers/profiling.md

```diff
@@ -5,7 +5,7 @@ about the timing of events, but instead provides combined information about the
 total time of execution of each function. It also produces a call graph that
 simplifies understanding the call relationship between functions.
 
-![Example part of execution graph](images/execution_graph.png)
+![Example part of execution graph ](images/execution_graph.png)
```

Comment:
I read up a little bit on profiling visualizations to try and understand this better, but it's new to me and I'd appreciate people with expertise checking for accuracy.

```suggestion
![A section of an example call graph showing the functions involved in a napari layerlist call. It has six connected boxes organized into three groups. First is the input: layerlist call. Second, the layerlist breaks into _extent_world and _step_size functions that turn into their own <listcomp> sections upon completion. Third, the <listcomp> boxes converge to make the output of the initial layerlist call.](images/execution_graph.png)
```

Reply:
A shortened version of this alt text if the content of the graph is not important to understanding.

```suggestion
![A section of an example call graph showing the functions involved in a napari layerlist call.](images/execution_graph.png)
```

Reply:
I suggest going with the shortened one because I interpret the image here as a suggestion of the kind of thing we're going to be able to produce with profiling tools. The short description does a great job at doing that, whereas the long description contains details that are most irrelevant at this point in the document.